man/oci-runtime-tool-generate.1.md

```diff
@@ -317,6 +317,7 @@ read the configuration from `config.json`.
 **--template**=PATH
   Override the default template with your own.
   Additional options will only adjust the relevant portions of your template.
+  whether template is valid or not will not be checked, this may lead to finally generated config file can't work directly. Currently, users should guarantee template is valid themselves.
```

Comment:
@Mashimiao Thanks! Maybe word this as
"Templates are not validated for correctness, so the user should ensure that they are correct."